notes/programming-languages/python/packages/nltk.md

```diff
@@ -0,0 +1,76 @@
+# Python Language Overview
+
+### The `nltk` Module
+
+The `nltk` (Natural Language Tool Kit) module is a good introduction to some common Natural Language Processing (NLP) processes, including Sentiment Analysis, Named Entity Recognition, and document preprocessing. You can also download corpus collections with nltk for practice or to serve as training data for machine learning applications.
+
+Reference:
+
+  + http://www.nltk.org/
+  + http://www.nltk.org/book
+
+> NOTE: `nltk` supports Python 2.7 & 3.4, 3.5
+
+First install the package using pip, if necessary:
+
+```` sh
+# install requests
+pip install nltk
+````
+
+#### Evaluating sentiment using `nltk`
+
+One of the most widely used NLP techniques is Sentiment Analysis. One of the modules available from `nltk` is the Vader Sentiment Analyzer, a relatively simple, vocabulary-based tool for measuring sentiment.
+
+```shell
+PS C:\Users\Mike> python
```

Comment:
consider converting this and similar`shell` code blocks into `python` blocks, and omitting the terminal call to the python console. this makes the code block more readable and understandable, IMHO.